src/lang/gettext_string_hash.hpp

```diff
@@ -0,0 +1,69 @@
+//
+// Created by machaviprace on 22.07.21.
+//
+#pragma once
+
+#include <stdio.h>
+#include <stdint.h>
+
+#define HASHWORDBITS 32
+
+//template <uint32_t TEST>
+class CTracker {
```

Comment:
I assume this is some kind of profiling code unrelated to gettext_string_hash, please put is into a separate file